proto/proto-backplane-grpc/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN set -eux; \
     /includes/deephaven/proto/session.proto \
     /includes/deephaven/proto/table.proto \
     /includes/deephaven/proto/application.proto \
-    /includes/deephaven/proto/inputtable.proto; \
+    /includes/deephaven/proto/inputtable.proto \
+    /includes/grpc/health/v1/health.proto; \
```

Comment:
at one point, I had it set up so it would sync all protofiles into an image that has grpcurl installed, and (at that time), I had to add it here to be able to run health check grpcurl from command line (in the days of the kubernetes impl).  

I could probably safely remove it and if I ever need to run grpcurl again, find a better way to bake it into the image-that-needs it.